src/Contract/Numeric/BigInt.purs

```diff
@@ -0,0 +1,12 @@
+-- | Extra functions for `JS.BigInt`.
+module Contract.Numeric.BigInt (abs) where
+
+import Prelude
+
+import JS.BigInt (BigInt)
+import JS.BigInt as BigInt
+
+abs :: BigInt -> BigInt
+abs x = case compare x (BigInt.fromInt 0) of
+  LT -> negate x
+  _ -> x
```

Comment:
This is in Prelude. Please remove this module entirely.

https://pursuit.purescript.org/packages/purescript-prelude/6.0.1/docs/Data.Ord#v:abs